runtime/rasdump/javadump.cpp

```diff
@@ -994,6 +997,11 @@ JavaCoreDumpWriter::writeEnvironmentSection(void)
 	_OutputStream.writeCharacters(_VirtualMachine->memoryManagerFunctions->omrgc_get_version(_VirtualMachine->omrVM));
 	_OutputStream.writeCharacters("\n");
 
+#if defined(OPENJDK_SHA) && defined(OPENJDK_TAG)
+	/* Write the OpenJDK Git SHA and build tag */
+	_OutputStream.writeCharacters("1CIOPENJDK " OPENJDK_SHA " based on " OPENJDK_TAG "\n");
```

Comment:
Similarly to the other comment about hard coding OpenJDK, I don't think it should be hard coded here. We could pick a more generic code, like "JDKEXT", or something else.

Reply:
Another suggestion is "JCL"